src/useFetch.ts

```diff
@@ -114,9 +110,8 @@ function useFetch<TData = any>(...args: UseFetchArgs): UseFetch<TData> {
         newRes = await fetch(url, options)
         res.current = newRes.clone()
 
-        if (cachePolicy === CACHE_FIRST) {
-          cache.set(response.id, newRes.clone())
-          if (cacheLife > 0) cache.set(response.ageID, Date.now())
+        if (persist || cachePolicy === CACHE_FIRST) {
```

Comment:
Hmm... I think having just the `cachePolicy` check here should work. If `persist` is set with the incorrect `cachePolicy` an Invariant Violation will already be thrown.

Reply:
I agree. I originally had a different idea about the persistence, but you're right.